index.html

```diff
@@ -126,6 +127,153 @@ <h3 class="heading-secondary heading-light">Fine ingredients</h3>
       </div>
     </section>
 
+    <section class="menu-section">
+      <div class="menu-columns">
+        <div>
+          <section>
+            <h4 class="heading-secondary heading-dark">Appetisers</h4>
+            <img src="gallery/decoration/decoration-2.png" class="decoration-menu" alt="decorative element">
+            <article>
+              <div class="menu-items">
+                <p>Tzatsiki</p>
+                <p>3.99</p>
+              </div>
+              <div class="line-decoration"></div>
+              <p class="description-of-menu-items">Refreshing traditional cucumber and garlic yoghurt
```

Comment:
Checkout the pattern logic:
```css
.block {
  .block-heading {}

  .block-items {
    .block-items-description {}

    .block-items-image {}
  }
}

```
```suggestion
              <p class="menu-items-description">Refreshing traditional cucumber and garlic yoghurt
```

`.description-of-menu-items` looks not reusable and not composable

Reply:
There are still doubts about the implementation of the paragraph of the correct structure. I am not closing this comment.